shared/src/main/java/uk/gov/di/authentication/shared/services/DynamoMfaMethodsService.java

```diff
@@ -87,4 +89,32 @@ private List<MfaMethodData> getMfaMethodsForNonMigratedUser(
             return List.of();
         }
     }
+
+    @Override
+    public MfaMethodData postMfaMethod(String email, MfaMethodCreateRequest request)
+            throws InvalidPriorityIdentifierException {
+        if (request.mfaMethod().priorityIdentifier() == PriorityIdentifier.DEFAULT) {
+            throw new InvalidPriorityIdentifierException(
+                    "Priority identifier for newly added MFA method should be BACKUP");
+        }
+
+        if (request.mfaMethod().method() instanceof SmsMfaDetail smsMfaDetail) {
+            dynamoService.addMFAMethodSupportingMultiple(
+                    email,
+                    new SmsMfaData(
+                            smsMfaDetail.phoneNumber(),
+                            true,
+                            true,
+                            request.mfaMethod().priorityIdentifier(),
+                            HARDCODED_SMS_MFA_ID));
```

Comment:
Here we don't have to hard-code, we can generate a new UUID. The hard-coding was to fix stuff in read.